quinzena1/projeto/projetos.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <link rel="stylesheet" href="style.css">
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Serviços</title>
+</head>
+<body>
+    <header class="header-cabeçalho">
```

Comment:
Evite qualquer caractere especial em nomes de classes (ou qualquer outro nome que você for dar pra algo em uma linguagem de programação).
Além disso, `header` e `cabecalho` são a mesma palavra, somente em linguas diferentes, então não seria necessaria a repeticao :)